eta/core/image.py

```diff
@@ -786,6 +786,17 @@ def from_dict(cls, d):
         return cls(images=images, schema=schema)
 
 
+class _BigImageLabelsSet(BigSet, _ImageLabelsSet):
+    '''Internal set type used by BigImageSetLabels to store labels.'''
+
+
+class BigImageSetLabels(ImageSetLabels):
```

Comment:
All the code finally pays off! Pretty slick that this is all we have to write to get a `Big` version of the class!